src/lib/sedlexing.mli

```diff
@@ -48,6 +48,14 @@ val create: (int array -> int -> int -> int) -> lexbuf
         position [pos], and return the number of characters provided. A
         return value of 0 means end of input. *)
 
+val set_curr_p: lexbuf -> Lexing.position -> unit
+    (** set initial [lex_curr_p] for [lexbuf],
+        if set to [Lexing.dummy_pos], Sedlexing would not track position info for you *)
+
+val set_filename: lexbuf -> string -> unit
+    (** set [lex_curr_p.pos_fname] for [lexbuf],
+        this implies to track the position info *)
```

Comment:
Try to start documentation comments with the function. For example here it would be: `[set_filename lexbuf file] sets <bla> to <bla>.`